yarn.lock

```diff
@@ -6292,11 +6292,16 @@ domhandler@^3.0.0, domhandler@^3.2.0:
   dependencies:
     domelementtype "^2.0.1"
 
-dompurify@^2.0.12, dompurify@^2.0.7:
+dompurify@^2.0.12:
```

Comment:
Is 2.0.12 being used by another pkg? Should we add a resolutions for 2.2.3?

Reply:
Let me look around